draft-ietf-rats-corim.md

```diff
@@ -1491,36 +1491,97 @@ and how the verifier matches a Reference Value against the Accepted Claims Set
+the processing of other groups.
+
+#### Matching a Conditional Endorsement Triple against the Accepted Claims Set
+
+A Conditional Endorsement Triple {{sec-comid-triple-cond-end}} is an efficient
+encoding of a Reference Value and an Endorsed Value.
+
+The triple's `stateful-environment-record` field contains both the `environment-map` and `measurement-map` parts of the Reference Value.
+
+The Endorsed Value is encoded using the same `environment-map`, plus the
+`measurement-value-map` field within the Conditional Endorsement Triple.
+
+For each Conditional Endorsement Triple, the verifier SHALL attempt to match the
+Reference Value against entries in the Accepted Claims Set (see {{sec-match-one-ref-val}}).
+If it matches then the verifier SHALL add the Endorsed value to the
+Accepted Claims Set (see {{sec-add-to-acs}}).
+
+>> *TODO: Is a Conditional Endorsement Triple affected by `reference-triple-record`s
+in the same group? I think not. Does this need to be mentioned?*
+
+#### Matching a Conditional Endorsement Series Triple against the Accepted Claims Set
+
+A Conditional Endorsement Series Triple {{sec-comid-triple-cond-series}} is an efficient
+encoding of a series of alternative Reference Values, each with a corresponding
+Endorsed Value. The Verifier adds the Endorsed Values corresponding to the first
+Reference Value which matches.
+
+All Reference Values and Endorsed Values in the triple use the same `environment-map`
+which is encoded within the `stateful-environment-record` part of the triple.
+
+Reference Values are encoded in two parts of this triple.
+
+Reference Values which apply to every Endorsed Value are encoded in the
+`stateful-environment-record`, in the same way as for the
+Conditional Endorsement Triple. If any of these Reference Values
+do not match Evidence or Endorsements in the Accepted Claims Set then then the
+entire triple is silently ignored.
+
+Reference Values which apply to only one Endorsed Value are encoded, with their
+matching Endorsed Value, in an array of `conditional-series-record`s.
+Each `conditional-series-record` encodes the `measurement-value-map` part for its
+Reference Values and for the corresponding Endorsed Value.
+
+The entries in the `conditional-series-record` array are processed in order. The verifier
+SHALL attempt to match the Reference Value part of each entry against entries
+in the Accepted Claims Set (see {{sec-match-one-ref-val}}).
+If the Reference Value matches then the verifier SHALL add the corresponding
+Endorsed Value to the Accepted Claims Set (see {{sec-add-to-acs}}) and
+SHALL skip processing the remaining entries in the array.
 
-Each group is processed independently of other groups. If a group fails to match
-the Accepted Claims Set then this does not affect the processing of other groups.
+>> *TODO: Is a Conditional Endorsement Series Triple affected by
+`reference-triple-record`s in the same group? I think not.
```

Comment:
It could be if the stateful-environment-record mentions endorsement claims that were added via a reference triple.

Reply:
I agree that the reference value part of Conditional Endorsement Series Triple could match against an endorsement added by an earlier group.
I guess what I was asking is whether CEST is in a group or not. If it's in a group then a mismatched reference value in that group could stop it from being processed. That seems odd to me, so I would rather say that CEST is not in a group, but I want to make sure everyone is happy with what I'm writing.